httpstats/transport_test.go

```diff
@@ -11,40 +11,46 @@ import (
 )
 
 func TestTransport(t *testing.T) {
-	backend := &stats.EventBackend{}
-	client := stats.NewClient(backend)
-	defer client.Close()
+	for _, transport := range []http.RoundTripper{
+		&http.Transport{},
```

Comment:
Technically `http.DefaultClient.Transport` is `nil` but yeah that's a good call just in case it changes.